api/api.proto

```diff
@@ -591,14 +591,23 @@ message CapacityUsageInfo {
 
 // A SdkStoragePolicy represents minimum set of volume specs to be 
 // follow while creating volumes.
-// If enforce in OpenStoragePolicy service, VolumeSpecPolicy will be 
-// used before creating volume to ensure minimum volume creation rules
-// are enforced.
+// If storage policy is set default in OpenStoragePolicy service, VolumeSpecPolicy will be 
+// used before creating volume to validate volume specs or ensure minimum volume creation 
+// rules followed
 message SdkStoragePolicy {
   // Name of storage policy.
   string name = 1;
   // VolumeSpecs to apply while creating volume.
   VolumeSpecPolicy policy = 2;
+  // Force if set to true volume specs will be overwritten, otherwise
+  // volume creation will be failed if volSpecs is not inline with storage policy
+  bool force = 3;
+  // AllowUpdate if set to true volume can be updated without storage Policy
```

Comment:
Can't use `AllowUpdate` because that is golang only. In other languages it can be `allow_update`. So in the comment, you do not need to refer to the name. You can just say: "Used to determine if a policy... and so on"